nav-app/src/app/help/help.component.scss

```diff
@@ -1,47 +1,54 @@
 @import "../../colors.scss";
 
 .help-dialog {
-    font-size: 11pt;
-    padding: 25px;
+  font-size: 11pt;
+  padding: 25px;
 
-    h1 {
-        text-align: center;
-        border-bottom: 1px solid black;
-    }
-
-    .content {
-        overflow-y: scroll;
-        max-height: 60vh;
-
-        .section-link {
-            cursor:pointer;
-            color:blue;
-            text-decoration:underline;
+  .table-of-contents {
+    list-style: none;
+    margin: 0;
+    padding: 0 25px 0 0;
+    li.toc-heading {
+      margin-bottom: 0;
+      a {
+        cursor: pointer;
+        color: blue;
+        text-decoration: underline;
+      }
+      @for $level from 1 through 6 {
+        &.level-#{$level} {
+          padding-inline-start: #{40 * ($level)}px;
         }
+      }
     }
-    
-    .close-button {
-        margin-top: 25px;
-        text-align: center;
-    }
-}
+  }
+  h1 {
+    text-align: center;
+    border-bottom: 1px solid black;
+  }
 
-// table of contents
-.toc {
-    list-style: none;
-    ul {
-        list-style: none;
+  .content {
+    overflow-y: scroll;
+    max-height: 60vh;
```

Comment:
Adding `padding-right: 25px;` to the content class helps separate the scrollbar from the text, increasing readability.

### Before
![Screen Shot 2021-07-13 at 10 10 02 AM](https://user-images.githubusercontent.com/28868888/125467143-6bc83db3-2e9c-477e-89de-6dce0c99e059.png)

### After
![Screen Shot 2021-07-13 at 10 10 15 AM](https://user-images.githubusercontent.com/28868888/125467187-05da8bc4-cb19-41ab-8063-f05ecd92346e.png)

Removing the gutter of the scrollbar may also improve the UI (though it's not always easy to do that in all browsers). [Here's one way to do that](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/develop/app/src/style/theme.scss#L7-L15).

Reply:
This padding also needs to get propagated to the layer format document UI -- the tables there are especially strange looking without the padding.

It may be worth extracting these rendered-markdown styles into an abstract class (`content { @extends %markdown-common; }` or common class (`.rendered-markdown` in the template) in a separate file to reduce code reuse.

Reply:
Note: scrollbar gutters should ideally only be removed for these dialogs, not across the application. I think the Layer box itself would look weird without the gutters.